src/main/java/org/entur/lamassu/metrics/MetricsService.java

```diff
@@ -117,6 +127,10 @@ private AtomicInteger getMissingRequiredFilesCounter(FeedProvider feedProvider)
     );
   }
 
+  private AtomicInteger getOverdueFilesCounter(FeedProvider feedProvider) {
+    return getCounter(feedProvider, overdueFilesCounters, FILES_OVERDUE);
```

Comment:
I think we probably want a gauge here not a counter?

Reply:
The suggested code replicates the [pre-existing naming inconsistency](https://github.com/entur/lamassu/blob/c0bb89803f370328abb252622481a2492eb89e40/src/main/java/org/entur/lamassu/metrics/MetricsService.java#L128-L139). From user/lamaussu perspective, the metrics are "counts", while in prometheus, as they are not strictly increasing, the correct metric is a `Gauge`. 

I adapted the current wording and think that should be ok.